### DISCUSSION TITLE
Fields and wide menu #2

### DISCUSSION THREAD

**Discussion by gegoune** [hrsh7th/nvim-cmp#891] [Q&A] [answered]
Slightly simplified version of [question from #1](https://github.com/hrsh7th/nvim-cmp/discussions/890) showing similar issue with just an abbr.

Default with `formatting = {}`:

<img width="165" alt="Screenshot 2022-04-12 at 21 39 54" src="https://user-images.githubusercontent.com/69750637/163031050-1240865d-36dc-459d-b8fd-56bb6fd6ac49.png">

So far so good. Seems like default is https://github.com/hrsh7th/nvim-cmp/blob/3192a0c57837c1ec5bf298e4f3ec984c7d2d60c0/lua/cmp/config/default.lua#L98
So let's try without kind and menu:

```lua
  formatting = {
    fields = { cmp.ItemField.Abbr  },
  },
```
and we get:

<img width="159" alt="Screenshot 2022-04-12 at 22 07 52" src="https://user-images.githubusercontent.com/69750637/163035584-3fe19f1c-3210-4f73-aa51-5edec79bcb06.png">

Not so good, window expands further than it really needs to. 
Why is abbr alone displayed like that? 

Also, should `field` be table of strings or enums from `cmp.ItemField`?

Thanks!

**Comment by siduck** [comment, marked as answer]
@gegoune its fixed now, update the plugin! https://github.com/hrsh7th/nvim-cmp/commit/28ab7f7d05db70acd88b06bb5830ec40805d8d7c
